examples/cli/main.go

```diff
@@ -32,34 +27,36 @@ var (
 )
 
 func main() {
-	id, err := identity.SignerFromMSPPath(*mspId, *mspPath)
-
-	if err != nil {
-		log.Fatalln(`Failed to load identity:`, err)
-	}
-
-	l, _ := zap.NewDevelopment()
-
-	core, err := client.NewCore(id, client.WithConfigYaml(*configPath), client.WithLogger(l))
-	if err != nil {
-		log.Fatalln(`unable to initialize core:`, err)
-	}
-
-	if err = core.Chaincode(*cc).Install(ctx, *ccPath, *ccVersion); err != nil {
-		log.Fatalln(err)
-	}
-
-	if err = core.Chaincode(*cc).Instantiate(
-		ctx,
-		*channel,
-		*ccPath,
-		*ccVersion,
-		*ccPolicy,
-		util.ToChaincodeArgs(*ccArgs),
-		prepareTransArgs(*ccTransient),
-	); err != nil {
-		log.Fatalln(err)
-	}
+	//  no instantiate option yet
+
+	//id, err := identity.SignerFromMSPPath(*mspId, *mspPath)
+	//
+	//if err != nil {
+	//	log.Fatalln(`Failed to load identity:`, err)
+	//}
+	//
+	//l, _ := zap.NewDevelopment()
+	//
+	//core, err := client.NewCore(id, client.WithConfigYaml(*configPath), client.WithLogger(l))
+	//if err != nil {
+	//	log.Fatalln(`unable to initialize core:`, err)
+	//}
+
+	//if err = core.Chaincode(*cc).Install(ctx, *ccPath, *ccVersion); err != nil {
+	//	log.Fatalln(err)
+	//}
+	//
+	//if err = core.Chaincode(*cc).Instantiate(
+	//	ctx,
+	//	*channel,
+	//	*ccPath,
+	//	*ccVersion,
+	//	*ccPolicy,
+	//	util.ToChaincodeArgs(*ccArgs),
+	//	prepareTransArgs(*ccTransient),
+	//); err != nil {
+	//	log.Fatalln(err)
+	//}
```

Comment:
uncomment and update along with new code.

Reply:
chaincode build with lscc and lifecycle will be in next commit